app/components/ui/revenueCard/cardComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
Please only import what you need from React, remove 

All instances of

import React from "react"

If you need HTMLAttributes import it alone as

import { HTMLAttributes, FC } from "react"